src/toplevel.lisp

```diff
@@ -250,7 +250,7 @@
    with-output-to-string with-package-iterator with-simple-restart
    with-slots with-standard-io-syntax write write-byte write-char
    write-line write-sequence write-string write-to-string y-or-n-p
-   yes-or-no-p zerop))
+   yes-or-no-p zerop override non-override clos))
```

Comment:
should the new symbols be exported from CL-USER instead of CL?

Reply:
```md
It works ?
Don't touch it
```
‎😃

Reply:
But CL is supposed to be the package of the standard CL.  That's why all implementations provide extra packages with extensions.

Reply:
The standard for declarations:  _**inline optimize type ftype notinline special**_  does not have a semantically close name for this _**non-standard declaration**_.

The form _**`(declaim (jscl::clos jscl::override))`**_  more likely?